src/services/discord/loggers/reactionAddLogger.js

```diff
@@ -0,0 +1,22 @@
+const { main_server, channels, categories } = require('../info')
+
+module.exports = async (reaction, user) => {
+  if (reaction.message.guild.id === main_server) {
+    if (user.bot) return
+    if (reaction.message.partial) await reaction.message.fetch()
+    const message = reaction.message
+    //Checks complete.
+
+    // return when promo implemented
```

Comment:
Saving this for later?

Reply:
Yeah, have to figure out a way to specify which message to look for reactions for, its a niche and interesting situation, since the bot creates a "trial announcement" and waits for a reaction on that message before sending out the real announcement